exasol_integration_test_docker_environment/test/test_click_api_consistency.py

```diff
@@ -0,0 +1,106 @@
+import unittest
+from typing import List, Any, Tuple
+
+import click.types
+
+from exasol_integration_test_docker_environment.cli import commands
+from exasol_integration_test_docker_environment.lib import api
+import inspect
+
+import utils as itde_test_utils
+
+
+def is_click_command(obj: Any) -> bool:
+    return isinstance(obj, click.Command)
+
+
```

Comment:
General question, can we build this in a way, that we only have on implementation, which we can use in Container-Tool and here

Reply:
Hm, I don't think there is an easy solution. Currently the test are part of the itde package, (which is something we want to fix). With that, it's possible to invoke the test also from script-language-container-tools. But maybe in the future there are small adjustments in the future (special cases in both projects), and then we would need to two versions again. I would prefer to keep it as is.

Reply:
I tried to separate as much as possible. Kept only the tests itself.